yml/apdb.yaml

```diff
@@ -0,0 +1,2704 @@
+    "@id": "#DiaObject.yPSFluxPercentile05"
+    datatype: float
+    description: Value at the 5% percentile of the y band fluxes.
+    fits:tunit: nJy
+  - name: yPSFluxPercentile25
+    "@id": "#DiaObject.yPSFluxPercentile25"
+    datatype: float
+    description: Value at the 25% percentile of the y band fluxes.
+    fits:tunit: nJy
+  - name: yPSFluxPercentile50
+    "@id": "#DiaObject.yPSFluxPercentile50"
+    datatype: float
+    description: Value at the 50% percentile (median) of the y band fluxes.
+    fits:tunit: nJy
+  - name: yPSFluxPercentile75
+    "@id": "#DiaObject.yPSFluxPercentile75"
+    datatype: float
+    description: Value at the 75% percentile of the y band fluxes.
+    fits:tunit: nJy
+  - name: yPSFluxPercentile95
+    "@id": "#DiaObject.yPSFluxPercentile95"
+    datatype: float
+    description: Value at the 95% percentile of the y band fluxes.
+    fits:tunit: nJy
+  - name: yPSFluxMin
+    "@id": "#DiaObject.yPSFluxMin"
+    datatype: float
+    description: Minimum observed y band fluxes.
+    fits:tunit: nJy
+  - name: yPSFluxMax
+    "@id": "#DiaObject.yPSFluxMax"
+    datatype: float
+    description: Maximum observed y band fluxes.
+    fits:tunit: nJy
+  - name: yPSFluxStetsonJ
+    "@id": "#DiaObject.yPSFluxStetsonJ"
+    datatype: float
+    description: StetsonJ statistic for the y band fluxes.
+  - name: yPSFluxLinearSlope
+    "@id": "#DiaObject.yPSFluxLinearSlope"
+    datatype: float
+    description: Linear best fit slope of the y band fluxes.
+    fits:tunit: nJy/day
+  - name: yPSFluxLinearIntercept
+    "@id": "#DiaObject.yPSFluxLinearIntercept"
+    datatype: float
+    description: Linear best fit Intercept of the y band fluxes.
+    fits:tunit: nJy
+  - name: yPSFluxMaxSlope
+    "@id": "#DiaObject.yPSFluxMaxSlope"
+    datatype: float
+    description: Maximum slope between y band flux obsevations max(delta_flux/delta_time)
+    fits:tunit: nJy/day
+  - name: yPSFluxErrMean
+    "@id": "#DiaObject.yPSFluxErrMean"
+    datatype: float
+    description: Mean of the y band flux errors.
+    fits:tunit: nJy
+  - name: flags
+    "@id": "#DiaObject.flags"
+    datatype: long
+    nullable: false
+    value: 0
+    description: Flags, bitwise OR tbd.
+    mysql:datatype: BIGINT
+    ivoa:ucd: meta.code
+  - name: lastNonForcedSource
+    "@id": "#DiaObject.lastNonForcedSource"
+    datatype: timestamp
+    nullable: false
+    description: Last time when non-forced DIASource was seen for this object.
+    mysql:datatype: DATETIME
+  - name: nDiaSources
+    "@id": "#DiaObject.nDiaSources"
+    datatype: int
+    nullable: false
+    description: Total number of DiaSources associated with this DiaObject.
+  primaryKey:
+  - "#DiaObject.diaObjectId"
+  - "#DiaObject.validityStart"
+  indexes:
+  - name: IDX_DiaObject_validityStart
+    "@id": "#IDX_DiaObject_validityStart"
+    columns:
+    - "#DiaObject.validityStart"
+  mysql:engine: MyISAM
+  mysql:charset: latin1
+- name: SSObject
```

Comment:
These SSObject tables are out of date.  The correct ones are at https://jira.lsstcorp.org/browse/RFC-620 (already in felis YAML!) 

We might as well update baselineSchema.yaml at the same time.

Reply:
SSObject schema was replaced with the one from RFC-620, in both new `apdb.yaml` and `baselineSchema.yaml`.